webapp/templates/public/help/registration.html.twig

```diff
@@ -15,7 +15,6 @@
     <img src="{{ asset('images/help/register.png') }}" alt="Registrace" class="mb-3 rounded-3 img-fluid"
          style="max-height: 400px">
     <p>
-        Následně se svými údaji <a href="/login">přihlaste</a>. V případě, že zapomenete heslo, napište nám na
-        <a href="mailto:example@delta-skola.cz">example@delta-skola.cz</a> a my vám ho resetujeme.
+        Následně se svými údaji <a href="/login">přihlaste</a>.
```

Comment:
umime tady generovat dynamicky url podle route name?

Reply:
jak to myslis? jak by se url mela menit?

Reply:
myslis takhle (viz commit)?